ios/use_react_native-0.68.rb

```diff
@@ -20,6 +14,29 @@ def include_react_native!(options)
     ENV['RCT_NEW_ARCH_ENABLED'] = '1'
   end
 
+  return unless new_arch_enabled
+
+  Pod::UI.warn(
+    'As of writing, TurboModule is still experimental and subject to change. ' \
+    'For more information, please see ' \
+    'https://reactnative.dev/docs/next/new-architecture-app-modules-ios.'
+  )
+  # At the moment, Fabric and TurboModule code are intertwined. We need to
+  # enable Fabric for some code that TurboModule relies on.
```

Comment:
really? I thought they were independent - is this something that we bubbled to Meta?

Reply:
I think they know, and probably the reason why the only have the one flag instead of two.